web/client/themes/default/less/dropdown-menu.less

```diff
@@ -92,6 +92,8 @@
 
 #mapstore-burger-menu {
     .dropdown-menu {
-        z-index: 1040;
+		z-index: 1040;
+		overflow-y: scroll;
```

Comment:
Is it possible to use overflow-y auto instead of scroll?
with overflow-y scroll the scrollbar is visible even when the menu height is less than 80% of the viewport height
![image](https://user-images.githubusercontent.com/19175505/106620517-ef928980-6571-11eb-8f8f-f66d5a725a53.png)

Reply:
sure, I set it "auto"